platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -102,6 +109,32 @@
 
   private String foreground;
 
+  public void setSourceKeyword(ResourceBundleLocator resourceBundleLocator) {
+    try {
+      ResourceBundle resourceBundle = resourceBundleLocator.getBundle(LANDING_PAGE_BASE_NAME);
+
+      if (resourceBundle.containsKey("dataSource")) {
+        this.sourceKeyword = (String) resourceBundle.getObject("dataSource");
+      } else {
+        this.sourceKeyword = "Data Source";
```

Comment:
✏️ This is exactly why my preference hen dealing with resource bundle is that a default resource bundle be provided as a class that ends up acting as a catch all and translations be defined in properties file. This prevents having to write boiler plate code like this for falling back. With that, you no longer need to do a `containsKey()` first. you simply know bundles will throw that `MissingResourceException` and by design, you know it cannot happen.